common/tests/fixtures/tron/sign_tx.json

```
@@ -267,6 +267,156 @@
+          "fee_limit": 100000000,
+          "timestamp": 1770837350109
+        },
+        "contract": {
+          "_message_type": "TronTriggerSmartContract",
+          "data": "095ea7b3000000000000000000000000f2cd810c48c401d392ead3c6e1e1cb9f57750a580000000000000000000000000000000000000000000000000000000000000000",
+          "owner_address": "41f2cd810c48c401d392ead3c6e1e1cb9f57750a58",
+          "contract_address": "4142a1e39aefa49290f2b3f9ed688d7cecf86cd6e0"
+        },
+        "raw_data_hex": "0a02ee382208fdc0ba410e4ca5f4408091d3f1c4335aae01081f12a9010a31747970652e676f6f676c65617069732e636f6d2f70726f746f636f6c2e54726967676572536d617274436f6e747261637412740a1541f2cd810c48c401d392ead3c6e1e1cb9f57750a5812154142a1e39aefa49290f2b3f9ed688d7cecf86cd6e02244095ea7b3000000000000000000000000f2cd810c48c401d392ead3c6e1e1cb9f57750a58000000000000000000000000000000000000000000000000000000000000000070ddad86f1c433900180c2d72f",
+        "raw": {
+          "visible": false,
```

Biome (2.3.14)

[error] The key visible was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)